nlp/index.js

```diff
@@ -0,0 +1,264 @@
+/*
+ * Copyright (c) AXA Group Operations Spain S.A.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining
+ * a copy of this software and associated documentation files (the
+ * "Software"), to deal in the Software without restriction, including
+ * without limitation the rights to use, copy, modify, merge, publish,
+ * distribute, sublicense, and/or sell copies of the Software, and to
+ * permit persons to whom the Software is furnished to do so, subject to
+ * the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be
+ * included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+ * LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+ * WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+const fs = require('fs')
+const { BuiltinMicrosoft } = require('@nlpjs/builtin-microsoft')
+const { BuiltinDuckling } = require('@nlpjs/builtin-duckling')
+const { containerBootstrap } = require('@nlpjs/core-loader')
+const { Language } = require('@nlpjs/language')
+const { LangAll } = require('@nlpjs/lang-all')
+const { Nlp } = require('@nlpjs/nlp')
+const { Evaluator, Template } = require('@nlpjs/evaluator')
+const { fs: requestfs } = require('@nlpjs/request')
+const SentimentManager = require('./sentiment')
+
+class NlpManager {
+  constructor (settings = {}) {
+    this.settings = settings
+    if (!this.settings.container) {
+      this.settings.container = containerBootstrap()
+    }
+    this.container = this.settings.container
+    this.container.registerConfiguration('ner', {
+      entityPreffix: '%',
+      entitySuffix: '%'
+    })
+    this.container.register('fs', requestfs)
+    this.container.register('Language', Language, false)
+    this.container.use(LangAll)
+    this.container.use(Evaluator)
+    this.container.use(Template)
+    this.nlp = new Nlp(this.settings)
+    this.sentimentManager = new SentimentManager()
+    if (this.settings.ner) {
+      if (this.settings.ner.ducklingUrl || this.settings.ner.useDuckling) {
+        const builtin = new BuiltinDuckling(this.settings.ner)
+        this.container.register('extract-builtin-??', builtin, true)
+      } else {
+        const builtin = new BuiltinMicrosoft(this.settings.ner)
+        this.container.register('extract-builtin-??', builtin, true)
+      }
+    } else {
+      const builtin = new BuiltinMicrosoft(this.settings.ner)
+      this.container.register('extract-builtin-??', builtin, true)
```

Comment:
This `BuiltinDuckling` and `BuiltinMicrosoft` seem to be mutually exclusive, are we using both?
Or could we potentially get rid of one of them?

Reply:
Took a look at it, seems like we're using the default microsoft one which is way way larger than the other one, so getting rid of the other would probably not really improve much.

Reply:
I just tested this, and we can actually get rid of both!

Reply:
pushed. 
Storage also went down by another 20 mb
![after2](https://user-images.githubusercontent.com/41830515/99305808-4d200f80-287a-11eb-9160-9b7136897107.png)

Reply:
Docker Image size is also quite a bit down (to `415MB`)
So all in all this brings it down from `455MB` to `415MB`

Reply:
Wait no, memory usage also improved a bit:

<img width="488" alt="Screenshot 2020-11-16 at 22 34 04" src="https://user-images.githubusercontent.com/13718901/99310811-dbd16400-285b-11eb-8900-46f1659c2368.png">